<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="plot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>891</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Channel No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Resolution (ns)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Channel_comboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Res_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Plot Controls</string>
    </property>
    <widget class="QPushButton" name="plot_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="log_pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y axis Log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Clear Plot</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fitting Controls</string>
    </property>
    <widget class="QComboBox" name="FittingFunc_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fitting function</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fit_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Get Lifetime!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="config_fit_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Configure Fit Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="Result_textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>256</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Export Plot Settings</string>
    </property>
    <widget class="QCheckBox" name="save_w_fit_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save with Fit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="export_plot_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Export Publication
Ready Figure!</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
